geoportal/c2cgeoportal_geoportal/scaffolds/create/Dockerfile

```diff
@@ -1,11 +1,5 @@
-FROM debian:stretch
-LABEL maintainer Camptocamp "info@camptocamp.com"
-
-RUN \
-  apt-get update && \
-  apt-get install --assume-yes --no-install-recommends gettext-base python3 && \
-  apt-get clean && \
-  rm --recursive --force /var/lib/apt/lists/*
+ARG VERSION
```

Comment:
is it working to not have the `FROM` on the first line ?

Reply:
Yes :-)

Reply:
Yes, but only with a recent version of docker

Reply:
It's released tow year ago... (from  17.05)